app/__init__.py

```diff
@@ -1,11 +1,41 @@
 """Initializes the flask app."""
-from flask import Flask
+from flask import Flask, render_template, redirect, session
+from forms import LoginForm
 from models import connect_db
 from config import app_config
+import secret
 
 app = Flask(__name__)
+app = Flask(__name__, static_folder='../static',)
```

Comment:
@ethanschreur why do we have two initialization of app.

Reply:
That is a mistake. which I will fix.